pyt/helper_visitors/label_visitor.py

```diff
@@ -288,3 +288,29 @@ def visit_Name(self, node):
 
     def visit_Str(self, node):
         self.result += "'" + node.s + "'"
+
+    def visit_joined_str(self, node, surround=True):
+        for val in node.values:
+            if isinstance(val, ast.Str):
+                self.result += val.s
+            else:
+                self.visit(val)
+
+    def visit_JoinedStr(self, node):
+        self.result += "f\'"
+        self.visit_joined_str(node)
+        self.result += "'"
+
+    def visit_FormattedValue(self, node):
+        self.result += '{'
```

Comment:
Nit: Maybe add abstract grammar docstrings, I [have](https://github.com/python-security/pyt/blob/5b1f2622ff86076fde196a14e6ad4f3e8d4db085/pyt/cfg/stmt_visitor.py#L207-L209) started [to do this recently](https://github.com/python-security/pyt/blob/5b1f2622ff86076fde196a14e6ad4f3e8d4db085/pyt/cfg/expr_visitor.py#L243-L253).

e.g.

```python
"""
    JoinedStr(expr* values)
"""
```

```python
"""
    FormattedValue(expr value, int? conversion, expr? format_spec)
"""
```